airflow/contrib/operators/file_to_gcs.py

```diff
@@ -37,6 +37,8 @@ class FileToGoogleCloudStorageOperator(BaseOperator):
     :type google_cloud_storage_conn_id: str
     :param mime_type: The mime-type string
     :type mime_type: str
+    :type gzip: Allows for file to upload as gzip
+    :param gzip: boolean
```

Comment:
Incorrect docstring, you have mixed `type` and `param`. Replace it to:

```
    :param gzip: Allows for file to upload as gzip
    :type gzip: boolean
```

Reply:
It would also be good to add more detail to this parameter and also at Line 28 that says that optionally you can also compress the file before uploading. An example, would be good.

https://github.com/apache/incubator-airflow/blob/e455e75107f84d5b47fa5b2cda1c68fb6016f0f4/airflow/contrib/operators/file_to_gcs.py#L28